server/jobs/index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+import fs from 'fs';
+
+const getJobs = (folderName) => {
+  const jobs = fs
+    .readdirSync(`${__dirname}/${folderName}`)
+    .map((fileName) => {
+      const { default: exported } = require(`${__dirname}/${folderName}/${fileName}`);
```

Comment:
What does `default` used here represent?

Reply:
It represents the default export from the module required.
I know you might be thinking that I could have just done:
```
const exported = require(`${__dirname}/${folderName}/${fileName}`);
```
The approach above won't work because the required file is making use of `es6` `export` syntax to create its default export.
You can check [here](https://hackernoon.com/import-export-default-require-commandjs-javascript-nodejs-es6-vs-cheatsheet-different-tutorial-example-5a321738b50f) for more info about this.

Reply:
I think the `exported` used here may not necessary since `default` gives you the value in `exported`

Reply:
What if you just did:
```
.map((fileName) => {
       const { default } = require(`${__dirname}/${folderName}/${fileName}`);
       return typeof default === 'function' ? default : () => {};
    });
Would it not work?

Reply:
`default` is a [keyword in javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/default), you can't just toss it around like that. That's the reason renamed it to `exported`